doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -117,6 +117,7 @@
 - Remove `algorithm::uniqueMergeSortedContainers` from `algorithm.hpp`
 - Move `std::string` conversion function to `iceoryx_dust` [\#1612](https://github.com/eclipse-iceoryx/iceoryx/issues/1612)
 - The posix call `unlink` is directly used in `UnixDomainSocket` [\#1622](https://github.com/eclipse-iceoryx/iceoryx/issues/1622)
+- Make iox::byte_t a distinct type [\#1900](https://github.com/eclipse-iceoryx/iceoryx/issues/1900)
```

Comment:
Please add a section under `API Breaking Changes:` which mentions the renaming of `byte_t` to `byte` and also that it isn't a simple type alias anymore.